src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -37,6 +37,8 @@ public LinuxContainerEventGenerator(IEnvironment environment, Action<string> wri
 
         public static string DetailsEventRegex { get; } = $"{ScriptConstants.LinuxFunctionDetailsEventStreamName} (?<AppName>[^,]*),(?<FunctionName>[^,]*),\\\\\"(?<InputBindings>.*)\\\\\",\\\\\"(?<OutputBindings>.*)\\\\\",(?<ScriptType>[^,]*),(?<IsDisabled>[0|1])";
 
+        public static string AzureMonitoringEventRegex { get; } = $"{ScriptConstants.LinuxAzureMonitoringEventStreamName} (?<Level>[0-6]),(?<ResourceId>[^,]*),(?<OperationName>[^,]*),(?<Category>[^,]*),(?<RegionName>[^,]*),(?<Properties>[^,]*)";
```

Comment:
Question -- do these regex's get used in the production code anywhere? Or just in tests? If just tests, should they move there?

Reply:
yes they get used in our diagnostics agent running on the service fabric node.  This is the regex it uses to filter functions host logs.